src/open_inwoner/openzaak/tests/test_case_detail.py

```diff
@@ -253,12 +267,12 @@ def _setUpMocks(self, m):
             json=paginated_response([self.status1, self.status2]),
         )
         m.get(
-            f"{ZAKEN_ROOT}rollen?zaak={self.zaak['url']}&omschrijvingGeneriek={RolOmschrijving.initiator}",
-            json=paginated_response([self.role]),
+            f"{ZAKEN_ROOT}rollen?zaak={self.zaak['url']}",
+            json=paginated_response([self.user_role, self.not_our_user_role]),
         )
         m.get(
-            f"{ZAKEN_ROOT}rollen?zaak={self.zaak['url']}&betrokkeneIdentificatie__natuurlijkPersoon__inpBsn={self.user.bsn}",
-            json=paginated_response([self.role]),
+            f"{ZAKEN_ROOT}rollen?zaak={self.zaak['url']}&omschrijvingGeneriek={RolOmschrijving.initiator}",
+            json=paginated_response([self.user_role]),
```

Comment:
Learning moment: requests.mocker matches on partial URLs in reverse order: 

I had to move the mock with both `zaak` and `omschrijving` below the one with only a `zaak` or it would always return the one with only `zaak` even if my request had the `omschrijving` parameter.